src/core/client/stream/tabs/Comments/Comment/flattenReplies/index.ts

```diff
@@ -0,0 +1,12 @@
+export const FlattenRepliesIndentLevel = 4 as const;
+
+export const replyIsFlattened = (
```

Comment:
I advocate starting a method name with a verb. wdyt of `isReplyFlattened` instead?